src/core/contrib.js

```diff
@@ -23,8 +23,18 @@ function findUserURLs(...thingsWithUsers) {
   return [...new Set(usersURLs)];
 }
 
+function getHeaders(conf) {
+  const headers = {};
+  const { githubUser, githubToken } = conf;
+  if (githubUser && githubToken) {
+    const credentials = btoa(`${githubUser}:${githubToken}`);
+    const Authorization = `Basic ${credentials}`;
+    Object.assign(headers, { Authorization });
+  }
```

Comment:
We should add support for token only authorization too, while we are here... so, something like:

```JS
if (!githubUser && githubToken) {
  const credentials = btoa(`${githubToken}`);
  const Authorization = `Token ${credentials}`;
  Object.assign(headers, { Authorization });
}
```

The above could be merged more cleanly with the current code.

Reply:
Ok yeah... this is all redundant. See:
https://github.com/w3c/respec/pull/2099/files#diff-fd8e14e6c335ae838f06d979287518d8R27

This should be going through that code path... or that code path should be refactored to handle this case too.